deepmd/main.py

```diff
@@ -737,6 +737,18 @@ def main_parser() -> argparse.ArgumentParser:
         default=None,
         help="Model branch chosen for changing bias if multi-task model.",
     )
+    parser_change_bias.add_argument(
+        "--skip-elementcheck",
+        action="store_true",
+        help="Enable this option to skip element checks if any error occurs while retrieving statistical data.",
+    )
+    parser_change_bias.add_argument(
+        "-mf",
+        "--min-frames",
+        default=10,
```

Comment:
This default value is a bit confusing since, by default, all data is used.

https://github.com/deepmodeling/deepmd-kit/blob/4ce9cfba9493f7dc256091814650ac4d38d971da/deepmd/main.py#L709-L715